lib/reporters/html/view-model.js

```diff
@@ -7,11 +7,12 @@ const TestCounter = require('../../test-counter');
 const NO_STATE = 'NO_STATE';
 
 module.exports = class ViewModel {
-    constructor(config) {
+    constructor(config, reporterOptions) {
         this._tree = {name: 'root'};
         this._skips = [];
         this._counter = new TestCounter();
         this._config = config;
+        this._reporterOptions = reporterOptions || {};
```

Comment:
I did this, because some tests for viewmodel failed and I also thought, that since without this option, there could be errors thrown, that i should have some default value for that case.